### PR TITLE
Be able to set class names for gauge.

### PR DESCRIPTION
**Pull request by HejdesGit**
I have added the ability to set the gauge class names.

We are using [BEM](http://getbem.com/introduction/) and our css-lint is giving us warnings for the class names.

I also added some git ignores.

### CODE REVIEW

**Comment by naikus** [comment]
Hi, I'm travelling and will be able to review this the day after.

**Comment by HejdesGit** [comment]
Sounds good.
Let me know if I should change something.

File: src/gauge.js [hunk 1]
```diff
@@ -213,7 +216,7 @@
         var gaugeElement = svg("svg", {"viewBox": "0 0 1000 1000", "class": "gauge"},
```
Comment (naikus) [inline src/gauge.js]: Lets have the the option for the gauge class too. That defaults to 'gauge'

Comment (HejdesGit) [inline src/gauge.js]: Good idea. Now fixed.

**Pull request merged**